src/patterns/components/stepper/step-description.hbs

```diff
@@ -10,7 +11,7 @@ order: 2
   aria-orientation="vertical">
   <li
     role="tab"
-    class="sprk-c-Stepper__step sprk-c-Stepper__step--selected sprk-c-Stepper__step--first"
+    class="sprk-c-Stepper__step sprk-c-Stepper__step--selected"
```

Comment:
Since we are changing the markup here for vanilla by adding this class with JS we now need to make sure that we aren't adding it twice. All existing vanilla stepper users will already have this class. Can you make sure that your new JS accounts for that?

Reply:
@bobwise  did you see this comment?